virtual_rainforest/models/soil/carbon.py

```diff
@@ -190,9 +247,14 @@ def convert_moisture_to_scalar(
     needed.
 
     Args:
-        soil_moisture: soil moisture for each soil grid cell
+        soil_moisture: relative water content for each soil grid cell (unitless)
     """
 
+    if any(soil_moisture > 1.0) or any(soil_moisture < 0.0):
+        log_and_raise(
```

Comment:
I think these have to be `np.any` to handle the array inputs.  Otherwise you get:

```python
In [8]: x = np.arange(24).reshape(4,6)

In [9]: any(x > 4)
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
Input In [9], in <cell line: 1>()
----> 1 any(x > 4)

ValueError: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()
```